README.md

```diff
@@ -129,26 +132,26 @@ or by using the toolbar.
 
 ### Commands
 
-Device Simulator Express provides several commands in the Command Palette (F1 or Ctrl + Shift + P/ Cmd + Shift + P for Mac OS) for working with \*.py files:
+Device Simulator Express provides several commands in the Command Palette (`F1` or `Ctrl + Shift + P` / `Cmd + Shift + `P for Mac OS) for working with \*.py files:

```

Comment:
The `P` in `Cmd + Shift + P` for Mac OS is outside the single quotes.

Reply:
ty for catching that :D